app/models/workflow_execution.rb

```diff
@@ -73,4 +73,26 @@ def as_wes_params
       workflow_url:
     }.compact
   end
+
+  def self.ransackable_attributes(_auth_object = nil)
+    %w[id run_id state created_at updated_at]
+  end
+
+  def self.ransackable_associations(_auth_object = nil)
+    %w[]
+  end
+
+  def self.metadata_sort(field, dir)
```

Comment:
Since this code matches the `self.metadata_sort(field, dir)` method in `sample.rb` can you make this into a model concern? The only differences within the 2 methods is `WorkflowExecution.arel_table[:metadata]` here and `Sample.arel_table[:metadata]` in `sample.rb`. You can just use `arel_table[:metadata]` in their place. Once the concern is created you can use it in the `workflow_execution` and `sample` models with `include CONCERN_NAME`

Reply:
51798ad0f7816687303bea8d43af54546f49dc4c